stack.yaml

```diff
@@ -14,7 +14,7 @@ extra-deps:
     https://gitlab.com/morley-framework/morley.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    4e55ff8d32549bb4a79f6d4d8c265d26d0bde572 # master
+    e6fa4c2b98dcd31ffafd307dc204cc937a070b99  # master
```

Comment:
Doesn't 4e55ff8d32549bb4a79f6d4d8c265d26d0bde572 work?

Reply:
Yes, it yet breaks the `morley-ledgers` :pepe_sad:

And oh, the new commit is actually not master, so we have to wait for the PR of Ilya.